src/main/groovy/life/qbic/portal/sampletracking/datasources/SampleTracker.groovy

```diff
@@ -1,6 +1,7 @@
 package life.qbic.portal.sampletracking.datasources
 
 import groovy.util.logging.Log4j2
+import io.micronaut.core.type.Argument
```

Comment:
Where does that `Argument` come from? What does it do? Let's have an argument... (sorry couldn't resist)

Reply:
That's what it does. It enables that a complex expected response object can be converted properly:
```
try {
                response = client.withCloseable { it.toBlocking().exchange(request,
                                                    Argument.of(List.class, Location.class))}
   }
```